test/puppetlabs/puppetdb/testutils/jetty.clj

```diff
@@ -263,8 +264,7 @@
                                                      (.setUseJmx broker# false))]
      (do ~@body)))
 
-(defmacro with-command-endpoint
-  "Invoke `body` with a different command endpoint (destination) name"
-  [new-endpoint-name & body]
-  `(binding [puppetlabs.puppetdb.cli.services/mq-endpoint ~new-endpoint-name]
-     (do ~@body)))
+(defn sync-command-post [base-url cmd version payload]
```

Comment:
Created a ticket for refactoring [jetty.clj](https://tickets.puppetlabs.com/browse/PDB-1255)